patterns/dxp/aspect_oriented_programming.md

```diff
@@ -0,0 +1,53 @@
+# Aspect Oriented Programming guide of use
+
+> **NOTE:** This plugin was originally in metal-plugins but now lives in portal so we can provide a better developing experience.
+
+AOP provides tools to improve control on what is executed before and after other method is invoked. You can finde this component in `frontend-js-web` module in portal.
```

Comment:
- "control on" → "control of"
- "other method" → "another method"
- "finde" → "find"
- "`frontend-js-web` module" → "the `frontend-js-web` module" (bonus points if you link it, in which case you can delete the "in portal" bit.